test/e2e/opaque_resource.go

```diff
@@ -38,7 +38,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = framework.KubeDescribe("Opaque resources [Feature:OpaqueResources]", func() {
+var _ = framework.KubeDescribe("Opaque resources [Serial]", func() {
```

Comment:
Alright. I think it was/is failing for other reasons. Once it passes we an try removing the serial tag.